bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -240,20 +249,6 @@ publishing {
                     connection.set("scm:https://github.com/mongodb/libmongocrypt")
                     developerConnection.set("scm:git@github.com:mongodb/libmongocrypt")
                 }
-                withXml {
```

Comment:
This modification is no longer needed due to using the declared version rather than the resolved version(s).